data-structures-stacks/max-value.js

```diff
@@ -0,0 +1,21 @@
+// I cheated on this one! 😏
+export default function countValues(stack) {
+  let max = -Infinity;
+  while (stack.peek() !== undefined) {
+    max = Math.max(max, stack.pop());
+  }
+  return max;
+}
+
+// export default function maxValue(stack) {
```

Comment:
This solution down here works, too, but it makes a copy of the array so it's not an ideal solution.

Reply:
Noted.